### DISCUSSION TITLE
Support only tailing since stern starts

### DISCUSSION THREAD

**Discussion by felipecrs** [stern/stern#203] [Q&A] [answered]
**What would you like to be added**:

Something like: `--since=now`, or perhaps `--since=0s` (which isn't currently supported).

**Why is this needed**:

I use stern in CI/CD, and I want to only show logs that appear AFTER I start stern.

Currently, the closest option is `--since=1s`.

**Comment by superbrothers** [comment, marked as answer]
You can use `stern --tail 0 <query>`.

```
$ stern -h | grep -- --tail
      --tail int                   The number of lines from the end of the logs to show. Defaults to -1, showing all logs. (default -1)
```

If you still have any issue, please feel free to reopen this issue.

**Reply by felipecrs** [reply]
Hm... I guess this makes sense. Thank you.

**Reply by felipecrs** [reply]
Thanks. This works. I would expect `--since=0s` to work the same way, by the way, but that's ok.

**Reply by felipecrs** [reply]
Another thing, you added the `stern -h` to your answer, but I hope you realized that the description of the flag hasn't anything that makes to understand that `--tail=0` would mean to only get new log lines.
